server/models/AvailabilityModel.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+  date: {
+    type: Date,
+    required: true,
+  },
+  startTime: {
+    type: String,
+    required: true,
+  },
+  endTime: {
+    type: String,
+    required: true,
+  },
```

Comment:
Don't think you are comprehending the availability, I'll go over this tomorrow in standup.